src/fairseq2/recipes/lm/instruction_finetune.py

```diff
@@ -54,16 +54,16 @@ class InstructionFinetuneConfig:
     """Holds the configuration of an instruction-finetuning recipe."""
 
     # Data
-    dataset_name: str = "openeft"  # TODO: fix!
+    dataset_name: str = "generic_instruction"
```

Comment:
`generic_instruction` is the family name. This should point to a dataset name though. Until we have some open source dataset, we can keep "openeft" as placeholder.

Reply:
ah true! will revert